pkg/asset/tls/certkey_test.go

```diff
@@ -0,0 +1,183 @@
+package tls
+
+import (
+	"bytes"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"io/ioutil"
+	"net"
+	"os"
+	"testing"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/types"
+)
+
+type fakeInstallConfig bool
+
+var _ asset.Asset = fakeInstallConfig(false)
+
+func (f fakeInstallConfig) Dependencies() []asset.Asset {
+	return nil
+}
+
+func (f fakeInstallConfig) Generate(map[asset.Asset]*asset.State) (*asset.State, error) {
+	return &asset.State{
+		Contents: []asset.Content{
+			{
+				Name: "fakeInstallConfig",
+				Data: []byte{},
+			},
+		},
+	}, nil
+}
+
+func TestCertKeyGenerate(t *testing.T) {
+	testDir, err := ioutil.TempDir(os.TempDir(), "certkey_test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(testDir)
+
+	root := &RootCA{rootDir: testDir}
+	rootState, err := root.Generate(nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var installConfig fakeInstallConfig
+	installConfigState, _ := installConfig.Generate(nil)
```

Comment:
It looks like you're ignoring an error here.  Can you replace this with:

```go
installConfigState, err := installConfig.Generate(nil)
if err != nil {
  t.Fatal(err)
}
```

Reply:
yeah cuz it's fake install config, but I added anyway.